variables.tf

```
@@ -32,6 +32,12 @@ variable "loadbalancer_crosszone" {
   description = "Configure the classic load balancer to route traffic evenly across all instances in all Availability Zones rather than only within each zone."
 }
 
+variable "loadbalancer_classic_connection_idle_timeout" {
```

You can omit the classic portion of the name to keep it consistent

```suggestion
variable "loadbalancer_connection_idle_timeout" {
```